src/main/java/rip/bolt/ingame/ranked/RankedManager.java

```diff
@@ -109,17 +112,43 @@ private void updateMatch(BoltMatch match) {
       return;
     }
 
-    List<MatchPlayer> players =
+    MatchManager mm = PGM.get().getMatchManager();
```

Comment:
this whole bit of logic could be moved elsewhere I see the `RankedManager` as something that interacts with the match managing its states. maybe another `RankManager` but for these stats?

Reply:
then once this method is complete it calls `notifyUpdates` inside a manager seems odd